platform-common-core/src/main/java/com/softicar/platform/common/core/CommonCoreI18n.java

```diff
@@ -5,6 +5,10 @@
 
 public interface CommonCoreI18n {
 
+	I18n0 DECIMAL_SEPARATOR = new I18n0("Decimal Separator")//
+		.de("Dezimaltrennzeichen");
+	I18n0 DIGIT_GROUP_SEPARATOR = new I18n0("Digit Group Separator")//
+		.de("Zifferngruppenseparator");
```

Comment:
I think this should be `Zifferngruppentrennzeichen`, for the sake of symmetry

Reply:
absolutely right, thx